{"timestamp":"2025-10-29T21:46:39.967854","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-10-29T21:46:39.968670","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/crime_data_export.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-10-29T21:46:40.387756Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T21:46:40.396180Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T21:46:40.416250Z","level":"info","event":"Current task name:export_crime_data","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T21:46:40.434334Z","level":"info","event":"Dag name:crime_data_export","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T21:46:40.393896","level":"warning","event":"/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py:471: DeprecationWarning: Using Connection.get_connection_from_secrets from `airflow.models` is deprecated.Please use `get` on Connection from sdk(`airflow.sdk.Connection`) instead\n  warnings.warn(\n","logger":"py.warnings"}
{"timestamp":"2025-10-29T21:46:40.515586","level":"info","event":"Connection Retrieved 'snowflake_conn'","logger":"airflow.hooks.base"}
{"timestamp":"2025-10-29T21:46:40.531847","level":"info","event":"Snowflake Connector for Python Version: 3.17.2, Python Version: 3.12.11, Platform: Linux-6.6.87.2-microsoft-standard-WSL2-x86_64-with-glibc2.36","logger":"snowflake.connector.connection"}
{"timestamp":"2025-10-29T21:46:40.532880","level":"info","event":"Connecting to GLOBAL Snowflake domain","logger":"snowflake.connector.connection"}
{"timestamp":"2025-10-29T21:46:42.888109","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"ProgrammingError","exc_value":"001006 (22023): 01c00ada-0002-5da7-0002-c79e00018602: SQL compilation error:\ninvalid parameter 'HEADER'","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1215,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/crime_data_export.py","lineno":41,"name":"export_crime_data"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/cursor.py","lineno":1142,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/errors.py","lineno":286,"name":"errorhandler_wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/errors.py","lineno":341,"name":"hand_to_other_handler"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/errors.py","lineno":217,"name":"default_errorhandler"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-10-29T21:46:42.891902","level":"error","event":"Airflow Task Failed\n    DAG: crime_data_export\n    Task: export_crime_data\n    Execution Date: None\n    Log: http://localhost:8080/dags/crime_data_export/runs/manual__2025-10-29T21%3A46%3A24.505860%2B00%3A00/tasks/export_crime_data?try_number=1\n    ","logger":"root"}
{"timestamp":"2025-10-29T21:46:42.893263Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T21:46:42.894548Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T21:46:42.896128Z","level":"info","event":"Task:<Task(PythonOperator): export_crime_data>","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T21:46:42.897008Z","level":"info","event":"Failure caused by 001006 (22023): 01c00ada-0002-5da7-0002-c79e00018602: SQL compilation error:","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T21:46:42.897579Z","level":"info","event":"invalid parameter 'HEADER'","chan":"stdout","logger":"task"}
